jwst/regtest/test_miri_image.py

```diff
@@ -108,3 +109,22 @@ def test_miri_image_stage3_catalog(run_pipelines, diff_astropy_tables):
 
     diff = diff_astropy_tables(rtdata.output, rtdata.truth, rtol=1e-4)
     assert len(diff) == 0, "\n".join(diff)
+
+
+@pytest.mark.bigdata
+def test_miri_image_wcs(run_pipelines, fitsdiff_default_kwargs):
+    rtdata = run_pipelines
+
+    # get input assign_wcs and truth file
+    output = "det_image_1_MIRIMAGE_F770Wexp1_5stars_assign_wcs.fits"
+    truth_file = rtdata.get_truth('truth/test_miri_image_stages/'+output)
+
+    # Open the output and truth file
+    im = ImageModel(output)
+    im_ref = ImageModel(truth_file)
```

Comment:
These should be `datamodels.open()` calls.  And it would be nice to avoid "ref" in variable names as it is ambiguous between truth files and CRDS reference files.  This is the reason to use "truth".